experiments/audio/exp1699_anneal.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 The Marin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Training config for the Marin-Audio (Yodas2 checkpoint) annealing experiment."""
+
+from experiments.audio.tokenize_mls_en import mls_en_data_config
+from experiments.audio.tokenize_emilia import emilia_english_mixture_config
+from experiments.anneal_config import AnnealConfig
+from experiments.defaults import default_anneal
+
+from marin.execution.executor import executor_main
+from marin.resources import TpuPodConfig
+
+SEQ_LEN = 4096
+BASE_BATCH_SIZE = 256
+BATCH_SIZE = 512
+BASE_LEARNING_RATE = 3e-3
+LEARNING_RATE = BASE_LEARNING_RATE * (BATCH_SIZE / BASE_BATCH_SIZE) ** 0.5
+NUM_ANNEAL_STEPS = 50000
+NUM_ANNEAL_TRAINING_TOKENS = int(NUM_ANNEAL_STEPS * BATCH_SIZE * SEQ_LEN)
+
+checkpoint_step = 190000
+checkpoint_path = f"gs://marin-us-central1/checkpoints/exp1699_marin_yodas2-b5edae/checkpoints/step-{checkpoint_step}"
```

Comment:
the marin thing to do is to import the step that produced `gs://marin-us-central1/checkpoints/exp1699_marin_yodas2-b5edae/` and then use `yoda_step.cd(f"checkpoints/step-{checkpoint_step}")`